src/main/java/net/tiagofar78/prisonescape/game/phases/Waiting.java

```diff
@@ -2,4 +2,19 @@
 
 public class Waiting extends Phase {
 
+    private boolean _clockStopped;
+    
+    public Waiting(boolean clockStopped) {
+        _clockStopped = clockStopped;
+    }
+
+    @Override
+    public Phase next() {
+        return new Ongoing(_clockStopped);
+    }
+
+    @Override
+    public boolean isClockStopped() {
+        return _clockStopped;
```

Comment:
Usually when using State desing pattern functions are expected to have already defined values instead of a variable. In this case it shoud return false because in the starting phase nothing is happening and there is no need to keep the game clock running.